config.json

```diff
@@ -122,6 +122,22 @@
                 "MAX_LIMIT": 1,
                 "WINDOW_SIZE": 1440
             }
+        },
+        {
+            "ID": "DIGARD",
+            "NAME": "Digard Network Test",
+            "TOKEN": "DIGA",
+            "RPC": "https://api.digard.network/ext/bc/sNUvM8CDbYbwi99ih238cbwyRiDYsX6GPKnFF1NzignfKfte4/rpc",
+            "CHAINID": 960006,
+            "EXPLORER": "https://testnet.avascan.info/blockchain/digard/home",
+            "IMAGE": "https://cdn.digard.io/digard.jpg",
```

Comment:
Would you like to use a PNG or image with a transparent background here? I am ok with both...